RedHatInsights/tasks-frontend
# chore(RHINENG-7054): adopt fec.config.js and tsconfig

Resolves: https://issues.redhat.com/browse/RHINENG-7054

To test: 
1.Run the application
2. Make sure there are no changes and the app functions as expected


This needs to be merged after: https://github.com/RedHatInsights/frontend-components/pull/2029
